src/components/post/Post.tsx

```diff
@@ -0,0 +1,37 @@
+import {createContext} from 'react'
+import './Post.css'
+import PostBody from './PostBody/PostBody';
+import PostInfoBar from './PostInfoBar/PostInfoBar';
+import PostReactions from './PostReactions/PostReactions';
+
+interface PostPropsTypes {
+    text: string;
+    author: string;
+    date: Date;
+    likesAmount: number;
+    comments: string[];
+}
+
+export const PostContext = createContext({text: "", author: "", date: new Date(), likesAmount: 0, 
+    comments: [""]
+});
+
+function Post(props: PostPropsTypes) {
+    return (
+        <PostContext.Provider value={props}>
+        <div className="Post">
+            <div className='PostInfoBar'>
+                <PostInfoBar></PostInfoBar>
```

Comment:
we usually use self closing tag when there is nothing inside:
```suggestion
                <PostInfoBar />
```

do it in all places you can